jenkins-dsl/cassandra_job_dsl_seed.groovy

```diff
@@ -107,13 +113,13 @@ def isSplittableTest(targetName) {
 
 def jdks(branchName, targetName) {
     if (branchName == 'trunk' || branchName ==~ /cassandra-5.\d+/) {
-        if (!targetName.endsWith('dtest-upgrade')) {
+        if (!targetName.startsWith('dtest-upgrade')) {
```

Comment:
needs fix. needs to keep including `jvm-dtest-upgrade` too

Reply:
fixed: https://github.com/apache/cassandra-builds/compare/781950472b20ad8d2dcb1ff8080727cfbc41cd92..0e94869c206a61ef7418460bf89fdde919b1008c